docs/kyma-environment-broker/03-05-authorization.md

```diff
@@ -0,0 +1,69 @@
+# Authorization
+
+The Kyna Environment Broker provides the following ways to authorize users.
+
+## Basic auth
+
+To access the Kyma Environment Broker endpoints with basic authorization enabled, you must specify an Authorization Basic token header:
+
+```
+Authorization: Basic YnJva2VyOkVVMmpLQmVKOGc=
+```
+
+>**NOTE**: The basic auth implementation is currently being replaced by the Oauth2 implementation and will be deprecated soon.
+
+## Oauth2
+
+The Kyma Environment Broker allows to authorize users using the Oauth authentication. It is using the [ApiRule](https://github.com/kyma-project/kyma/blob/master/docs/api-gateway-v2/06-01-apirule.md) to provide a VirtualService and [Oathkeeper Access Rules](https://www.ory.sh/docs/oathkeeper/api-access-rules) according to the details specified in the CR.
+To authorize with the Kyma Environment Broker, use an OAuth2 client registered through the Hydra Maester controller. Look for more information [here](https://github.com/kyma-project/kyma/blob/master/docs/api-gateway-v2/08-01-exposesecure.md#register-an-oauth2-client-and-get-tokens).
+
+To access the Kyma Environment Broker endpoints with OAuth2 authorization enabled, use the `/oauth` prefix. For example:
+
+```
+/oauth/v2/catalog
+```
+
+You must also specify an Authorization Bearer token header:
+
+```
+Authorization: Bearer {ACCESS_TOKEN}
+```
+
+### Access token
+
+Follow the tutorial to obtain a new access token.
+
+1. Create a OAuth2 Client:
```

Comment:
add first step about exporting necessary env variables

---

which namespace should I use?